src/math.ts

```diff
@@ -0,0 +1,23 @@
+import { generatePercentChanceToFlake, log, PERCENT_CHANCE_OF_SUCCESS } from './utils'
+
+const clonedMathRandom = Math.random;
+
+Math.random = function () {
+  if (generatePercentChanceToFlake() > PERCENT_CHANCE_OF_SUCCESS) {
+    if (generatePercentChanceToFlake() > PERCENT_CHANCE_OF_SUCCESS) {
+      // Doubly-rare flake: return 3.14159265359
+      log('Math.random flaking (rare!)');
+      return 3.14159265359;
+    }
+    log('Math.random flaking');
+    const unixStr = `${Date.now}`; // will actually be yesterday via enflake
```

Comment:
Wouldn't this be `'function now() { [native code] }'` instead?

Reply:
Oop yep I forgot to call it >~< this is what I get for thinking "I'll just finish everything quickly before I need to leave" in the morning lmao